ipatool

```diff
@@ -1267,11 +1268,20 @@ def pr_list_command(ctx):
                 continue
             if lnames & labels_neg:
                 continue
+        commit_statuses = []
+        for c in pr.commits():
+            for statuses in c.statuses():
+                commit_statuses.append(statuses.state)
+        status_result = [list(i) for j, i in groupby(commit_statuses, lambda a: a)]
```

Comment:
`groupby` groups sequential runs so this does not handle interleaved results properly and leads to incorrect results.
Instead I'd suggest to do:
```python
status_counts = {}
for status in c.statuses():
    if status.state not in status_counts:
        status_counts[status] = 1
    else:
        status_counts[status] += 1
```